.github/workflows/darwin.yaml

```diff
@@ -77,10 +73,9 @@ jobs:
               # Enable -Wconversion by hand as well, because it seems to not be
               # enabled by default in the Xcode config.
               #
-              # Disable -Wunguarded-availability-new because we internally use
-              # APIs we added after our deployment target version.  Maybe we
-              # should change the deployment target version instead?
-              run: xcodebuild -target "Matter" -sdk macosx OTHER_CFLAGS='${inherited} -Werror -Wconversion -Wno-unguarded-availability-new'
+              # Disable availability annotations, since we are not building against a system
```

Comment:
```suggestion
              # Disable availability annotations, since we are not building a system
```

Reply:
Done, and for other places in this file.